src/styles/errorpage.scss

```diff
@@ -0,0 +1,16 @@
+.error-page{
+    position: absolute;
+    top:50%;
+    left:50%;
+    transform: translate(-50%, -50%);
```

Comment:
can you think of another way using flex to center the items

Reply:
Given flex properties to center these items.